client/templates/tweet-column-middle.html

```diff
@@ -1,3 +1,7 @@
 <div layout="column" layout-padding layout-fill">
-    <ng-include src="'/templates/tweet.html'" ng-repeat="tweet in tweets | filter:{pinned:'!true'} | orderBy : sortByDate : true track by $index" ng-class="{disabled : tweet.deleted, blocked: tweet.blocked}"></ng-include>
+    <ng-include
+        src="'/templates/tweet.html'"
+        ng-repeat="tweet in getTweets() | filter:{pinned:'!true'} | orderBy : sortByDate : true track by $index"
```

Comment:
You're right, we also have to change the other columns too though - right now we have:
Left: Pinned ^ !WallPriority
Right: !Pinned ^ WallPriority
Middle: Pinned ⊕ WallPriority

When really it should probably be
Left: Pinned
Right: WallPriority && !Pinned
Middle: !WallPriority && !Pinned

Although actually I think we should probably have a better way of sorting tweets into columns, that makes mutual exclusivity much more intuitive